### ISSUE TITLE
Feature request: pre-labeled chip boundaries

### ISSUE THREAD

**Issue by mjgleason** [GeoBigData/nacho#6]
In this case, I'd like to be able to feed into nacho a geojson with pre-labeled chip boundaries that would then output chips to separate folders based on the labels.

**Comment by drwelby** [comment]
Yes, this ties into the concept of Canvas "virtual chips". I'd like to hear more about potential use cases for chips stored as vectors.

**Comment by drwelby** [comment]
Some use cases: 

- User digitizes chips in a UI, the vector output is used to write chips
- as above, but the ML framework can read the vectors and directly load the raster
